rulewerk-owlapi/src/main/java/org/semanticweb/rulewerk/owlapi/OwlAxiomToRulesConverter.java

```diff
@@ -91,13 +92,24 @@
  */
 public class OwlAxiomToRulesConverter implements OWLAxiomVisitor {
 
+	Skolemization skolemization = new Skolemization();
```

Comment:
I am never using private fields without need. You cannot access private members in tests.

Reply:
as the field is not final (because it is reinitialized in the constructor), its value can be changed by any class in the same package. This is not desired. Although the field cannot be modified in classes outside this package (i.e. by a user), a developer must keep this in mind when trying to understand the code while looking trough the call hierarchy. Making the field private and using a package-protected getter would make the code easier to understand and less prone to errors.